src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useContext } from 'react';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { TodosContext } from '../../contexts/TodosContext';
+
+export const TodoList: React.FC = () => {
+  const { filteredTodos } = useContext(TodosContext);
+
+  return (
+    <ul className="todo-list" data-cy="todoList">
+      {filteredTodos.length > 0 && filteredTodos.map(todo => (
```

Comment:
The condition `filteredTodos.length > 0` is unnecessary. If `filteredTodos` is an empty array, then the map function will not loop over anything and return an empty array, which is effectively the same as not rendering anything.